src/drt/src/db/tech/frConstraint.h

```diff
@@ -1220,8 +1222,8 @@ class frLef58CutSpacingTableConstraint : public frConstraint
 
  private:
   odb::dbTechLayerCutSpacingTableDefRule* db_rule_;
-  frCoord default_spacing_;
-  bool default_center2center_;
+  std::pair<frCoord, frCoord> default_spacing_;
```

Comment:
It would be nicer to use a small struct with meaningful names rather than a pair.